watchdog/README.md

```diff
@@ -215,7 +215,7 @@ Auto-scaling starts at 1 replica and steps up in blocks of 5:
 
 You can override the minimum and maximum scale of a function through labels.
 
-Add these labels to the deployment if you want to sacle between 2 and 15 replicas.
+Add these labels to the deployment if you want to scale between 2 and 15 replicas.
```

Comment:
This is a typo, but we generally encourage an issue so that a maintainer can turn it around quickly.

https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md